src/utils/reconcileRates.test.ts

```diff
@@ -0,0 +1,259 @@
+import { PriceCategories } from '@/pages/steps/AdditionalInformationStep/PriceInformation';
+import { Offer } from '@/types/Offer';
+import { Organizer } from '@/types/Organizer';
+import { reconcileRates } from '@/utils/reconcileRates';
+
+const uitpasOffer = {
+  mainLanguage: 'nl',
+  organizer: { labels: ['uitpas'] },
+} as Offer;
+
+describe('reconcileRates', () => {
+  it('can overwrite default values', () => {
+    const currentRates = [
+      {
+        name: { nl: 'Base Tarif' },
+        category: PriceCategories.BASE,
+        price: '',
+        priceCurrency: 'EUR',
+      },
+    ];
+
+    const newRates = [
+      {
+        name: { nl: 'foobar' },
+        category: PriceCategories.BASE,
+        price: '15',
+        priceCurrency: 'EUR',
+      },
+    ];
+
+    expect(reconcileRates(currentRates, newRates)).toMatchSnapshot();
+    expect(reconcileRates(currentRates, [])).toEqual([
+      {
+        name: { nl: 'Base Tarif' },
+        category: PriceCategories.BASE,
+        price: '0,00',
+        priceCurrency: 'EUR',
+      },
+    ]);
+  });
+
+  it('can format prices', () => {
+    const currentRates = [
+      {
+        name: { nl: 'foo' },
+        category: PriceCategories.BASE,
+        price: '5',
+        priceCurrency: 'EUR',
+      },
+      {
+        name: { nl: 'bar' },
+        category: PriceCategories.BASE,
+        price: 2500000,
+        priceCurrency: 'EUR',
+      },
+    ];
+
+    // @ts-ignore
```

Comment:
If you must ignore a ts issue. use `@ts-expect-error` instead. typescript will show error if the `@ts-expect-error` isn't needed anymore.